spec/occi/core/locations_spec.rb

```diff
@@ -0,0 +1,111 @@
+module Occi
+  module Core
+    describe Locations do
+      subject(:locs) { locations }
+
+      let(:locations) { Locations.new(uris: uris) }
```

Comment:
I believe there was a reason for this ... ages ago when `rubocop` first terrorized me into using `subject` in the first place. Now, I use it just for consistency. Although it is quite possibly pointless.